src/storage/src/hummock/store/version.rs

```diff
@@ -108,12 +112,60 @@ pub struct HummockReadVersion {
     committed: CommittedVersion,
 }
 
-#[expect(unused_variables)]
 impl HummockReadVersion {
+    pub fn for_test() -> HummockReadVersion {
```

Comment:
We may add a `#[cfg(any(test, feature = "test"))]` use import the `INVALID_VERSION_ID` inside the method